src/Mod/TechDraw/Gui/MDIViewPage.cpp

```diff
@@ -1216,7 +1216,7 @@ bool MDIViewPage::compareSelections(std::vector<Gui::SelectionObject> treeSel, Q
 
 void MDIViewPage::showStatusMsg(const char* s1, const char* s2, const char* s3) const
 {
-    QString msg = QString::fromUtf8(tr("Selected:");
+    QString msg = QString::fromUtf8(tr("Selected:"));
     msg += QString::fromUtf8(" ");
     msg.append(QObject::tr("%1.%2.%3")

```

Comment:
TravisCI is [throwing an error](https://travis-ci.org/FreeCAD/FreeCAD/jobs/481131056#L6010-L6017)
```
/home/travis/build/FreeCAD/FreeCAD/src/Mod/TechDraw/Gui/MDIViewPage.cpp:1219:37: error: no viable conversion from 'QString' to 'const char *'
    QString msg = QString::fromUtf8(tr("Selected:"));
                                    ^~~~~~~~~~~~~~~
/usr/include/qt4/QtCore/qstring.h:330:41: note: passing argument to parameter here
    static QString fromUtf8(const char *, int size = -1);
                                        ^
1 error generated.
```
Not sure how to fix this.

Reply:
Fixed